README.md

```diff
@@ -1,24 +1,43 @@
 # Go Rules
-This repo provides Golang build rules for the [Please](https://please.build) build system. 
 
-# Basic usage
+This repo provides [Go](https://go.dev/) build rules for the [Please](https://please.build) build system.
+
+Go is especially suited to writing command line tools and utilities. Binaries can be run with `plz run`, or used
+as tools for [custom rules](https://please.build/codelabs/genrule/#0).
+
+# Installation
+
+First, add the plugin to your Please repo. In `plugins/BUILD`:
 
-First add the plugin to your project. In `plugins/BUILD`:
 ```python
 plugin_repo(
     name = "go",
-    revision="vx.x.x",
+    revision = "vx.x.x", # A go-rules version
 )
 ```
 
-Then add the plugin config:
+Next, define a version of Go for the plugin to use. In `third_party/go/BUILD`:
+
+```python
+subinclude("///go//build_defs:go")
```

Comment:
Should we recommend

```
[parse]
preloadsubincludes = ///go//build_defs:go
```

(`plz plugin init` sets this up)

Reply:
Agreed.

The phases of `plz` initialization and implicit actions are barriers to entry.  Documentation should put users on the happy path/path of least resistance, not the path of maximal detail.  The detail needs to be explained someplace, but it shouldn't be in the readmes that get things up to speed for a user.  If people want to see what changed, `git diff` should be sufficient to see what happens when someone runs `plz plugin init go`, or whatever.  $0.02.